scrapers_next/ca/committees_test.py

```diff
@@ -0,0 +1,243 @@
+from spatula import HtmlListPage, URL, XPath, CSS, HtmlPage, SelectorError
+from openstates.models import ScrapeCommittee
+import re
+
+
+class ChooseType(HtmlPage):
+    def process_page(self):
+
+        xpaths = {
```

Comment:
since the others only differ on process_page but get the exact same input, I'm debating whether it makes more sense to do this by dispatching to process_page_type1, process_page_type2 functions vs. different classes, but I'll leave that up to you & let you decide which feels more natural to you while working on it.

Reply:
I tried to turn the classes into functions but could not get it to work properly (would call the function within the dictionary)